wiki/Rules/Short/en.md

```diff
@@ -1,5 +1,5 @@
-Simplified osu! Rules
-========================
+Simplified
```

Comment:
following the folder's name, it should be `Short` (otherwise change the folder's name)